### DISCUSSION TITLE
Spoiling

### DISCUSSION THREAD

**Discussion by GambitFPS** [facebook/docusaurus#6601] [Q&A]
Does docusaurus feature spoiling? Discord has a spoiler tag (||spoiler||) for example

**Comment by Josh-Cena** [comment]
No, write your own Remark plugin to do this. And again, please ask in the Discord server next time.

**Reply by slorber** [reply]
Not sure what "spoiling" means exactly in this context but maybe details/summary views can be useful 🤷‍♂️ 

https://docusaurus.io/docs/markdown-features#details
